doc/when-things-go-wrong.rst

```diff
@@ -190,7 +193,7 @@ what is going on.
   Anyway, if a your GUI application fails,
   you can run your application on the command line,
   i.e. ``./dist/my_gui``.
-  
+
```

Comment:
White-space only change, please restore the old content.

Reply:
Ok